frontend/src/Map/Map.tsx

```diff
@@ -33,22 +37,34 @@ function Map({ muralClick, murals }: IMapProps) {
   const geolocateStyle = {
     bottom: 30,
     right: 0,
-    padding: '10px'
+    padding: "10px",
   };
 
   const imgStyle = {
-    maxWidth: '200px',
-    maxHeight: '200px',
-    paddingBottom: '10px'
+    maxWidth: "200px",
+    maxHeight: "200px",
+    paddingBottom: "10px",
+  };
+
+  const geojsonTour = {
```

Comment:
want to take it further to pull all tours from the backend on startup? this could be done in App.tsx similarly to murals [here](https://github.com/hack4impact-mcgill/mu-map/blob/main/frontend/src/App/App.tsx#L84) or in another way you see fit